mmv1/third_party/terraform/acctest/vcr_utils.go

```diff
@@ -243,13 +282,35 @@ func initializeReleaseDiffTest(c resource.TestCase, testName string) resource.Te
 		}
 	}
 
+	c.Steps = InsertDiffSteps(c, tempFile, releaseProvider, localProviderName)
+
+	return c
+}
+
+// InsertDiffSteps inserts a new step into the test case that reformats the config to use the release provider - this allows us to see the diff
+// between the local provider and the release provider. for a certain test, this will be used to see the diff between the local provider and the release provider.
+func InsertDiffSteps(c resource.TestCase, tempFile *os.File, releaseProvider string, localProviderName string) []resource.TestStep {
```

Comment:
Nit: could pass in `c.Steps` to make it slightly more clear (ie. original list of steps is passed in, new set of steps is returned)